index.js

```diff
@@ -1,663 +1,246 @@
-                en: ['c','C'],
-                ru: ['с','С'],
-            }
-        },
-
-        {
-            '86':   {
-                en: ['v','V'],
-                ru: ['м','М'],
-            }
-        },
-
-        {
-            '66':   {
-                en: ['b','B'],
-                ru: ['и','И'],
-            }
-        },
-
-        {
-            '78':   {
-                en: ['n','N'],
-                ru: ['т','Т'],
-            }
-        },
-
-        {
-            '77':   {
-                en: ['m','M'],
-                ru: ['ь','Ь'],
-            }
-        },
-
-        {
-            '188':   {
-                en: [',','<'],
-                ru: ['б','Б'],
-            }
-        },
-
-        {
-            '190':   {
-                en: ['.','>'],
-                ru: ['ю','Ю'],
-            }
-        },
-
-        {
-            '191':   {
-                en: ['/','?'],
-                ru: ['.',','],
-            }
-        },
-
-        {
-            '38':   {
-                en: ['Up','Up'],
-                ru: ['Up','Up'],
-            }
-        },
-
-        {
-            '16':   {
-                en: ['Shift','Shift'],
-                ru: ['Shift','Shift'],
-            }
-        },
-    ],
-
-    [
-        {
-            '17':   {
-                en: ['Ctrl','Ctrl'],
-                ru: ['Ctrl','Ctrl'],
-            }
-        },
-        {
-            '91':   {
-                en: ['Win','Win'],
-                ru: ['Win','Win'],
-                
-            }
-        },
-        {
-            '18':   {
-                en: ['Alt','Alt'],
-                ru: ['Alt','Alt'],
-            }
-        },
-        {
-            '32':   {
-                en: [' ',' '],
-                ru: [' ',' '],
-            }
-        },
-        {
-            '18':   {
-                en: ['Alt','Alt'],
-                ru: ['Alt','Alt'],
-            }
-        },
-        {
-            '17':   {
-                en: ['Ctrl','Ctrl'],
-                ru: ['Ctrl','Ctrl'],
-            }
-        },
-        {
-            '37':   {
-                en: ['Left','Left'],
-                ru: ['Left','Left'],
-            }
-        },
-        {
-            '40':   {
-                en: ['Down','Down'],
-                ru: ['Down','Down'],
-            }
-        },
-        {
-            '39':   {
-                en: ['Right','Right'],
-                ru: ['Right','Right'],
-            }
-        },
-    ],
-]
-// import {keyboard} from './src/data.js';
+// eslint-disable-next-line import/extensions
+import keyboard from './src/data.js';
 
 let currentLang = localStorage.getItem('lang') || 'en';
 let cups = false;
 const input = document.createElement('textarea');
-const root = document.createElement('div'); 
+const root = document.createElement('div');
 root.id = 'root';
 input.id = 'keyboard-input';
 document.body.append(input);
 document.body.append(root);
 input.disabled = true;
```

Comment:
In our task we have a description of the way it works, like 

> del удаляет символ, стоящий после курсора.

but it's impossible to do, because we cannot move cursor carriage